### ISSUE TITLE
cop ColonMethodCall shouldn't forbid constructor like calls

### ISSUE THREAD

**Issue by markijbema** [rubocop/rubocop#425]
Ruby has the tradition of adding constructor-like methods to instantiate classes, for instance `Array()` to construct an instance of `Array`. Some libraries offer this as well, like [Nokogiri](https://github.com/sparklemotion/nokogiri). Their readme suggests using colons, and I do think it makes sense in this case, and should be excepted from the ColonMethodCall:

``` ruby
doc = Nokogiri::HTML(open('http://www.google.com/search?q=sparklemotion'))
```

Opinions?

My suggestion would be to allow method calls which start with a capital letter.
